server/routes/index.ts

```diff
@@ -12,9 +13,9 @@ export default function routes(services: Services): Router {
 
   const home = new HomeController(services.bookerService)
 
-  get('/', home.view())
+  get(paths.HOME, home.view())
 
-  router.use('/book-visit', bookingJourneyRoutes(services))
+  router.use(bookingJourneyRoutes(services))
```

Comment:
Not setting path to `/book-visit` when mounting this route handler because the full path is specified when mounting the individual page routes in `server/routes/bookVisit/index.ts`. Keeps the `paths` constant simpler (i.e. I couldn't figure out an alternative set up that didn't require storing both the full paths **and** all the segments 😅!)